participant/src/round2.rs

```diff
@@ -31,6 +37,28 @@ pub fn round_2_request_inputs(
     let signing_package: SigningPackage = serde_json::from_str(signing_package_json.trim())
         .map_err(|_| Error::MalformedSigningKey)?;
 
+    #[cfg(feature = "redpallas")]
+    {
+        logger.log("Enter the randomizer (hex string):".to_string());
+
+        let mut json = String::new();
+
+        input.read_line(&mut json).unwrap();
+
+        // TODO: fix error
```

Comment:
Does this need to be fixed as part of this story?

Reply:
thanks! forgot about it, fixed in 1e17731